x-pack/plugin/esql/src/main/java/generated/org/elasticsearch/xpack/esql/expression/function/scalar/conditional/IntersectsConstantConstantEvaluator.java

```diff
@@ -0,0 +1,45 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License
+// 2.0; you may not use this file except in compliance with the Elastic License
+// 2.0.
+package org.elasticsearch.xpack.esql.expression.function.scalar.conditional;
+
+import java.lang.Override;
+import java.lang.String;
+import org.elasticsearch.compute.data.Block;
+import org.elasticsearch.compute.data.BooleanVector;
+import org.elasticsearch.compute.data.Page;
+import org.elasticsearch.compute.operator.EvalOperator;
+
+/**
+ * {@link EvalOperator.ExpressionEvaluator} implementation for {@link Intersects}.
+ * This class is generated. Do not edit it.
+ */
+public final class IntersectsConstantConstantEvaluator implements EvalOperator.ExpressionEvaluator {
```

Comment:
Hmmm - `ConstantConstantEvaluator` - something is funny there.

Reply:
Perhaps I giggled a little bit too.

But seriously, it does seem like the right name, from the various combinations supported:

```
    @Evaluator(extraName = "LongLong")
    @Evaluator(extraName = "LongConstant")
    @Evaluator(extraName = "LongString")
    @Evaluator(extraName = "ConstantConstant")
    @Evaluator(extraName = "ConstantString")
    @Evaluator(extraName = "StringString")
```